.stylelintrc

```diff
@@ -2,6 +2,9 @@
     "extends": ["stylelint-config-standard"],
     "customSyntax": "postcss-less",
     "rules": {
+        "string-quotes": null,
+        "import-notation": null,
+        "comment-empty-line-before": null,
```

Comment:
available values for this rule are `"always"` and `"never"`. don't use `null`
https://stylelint.io/user-guide/rules/comment-empty-line-before/